lib/cartopy/crs.py

```diff
@@ -1652,15 +1654,16 @@ class Geostationary(_Satellite):
 
     """
     def __init__(self, central_longitude=0.0, satellite_height=35785831,
-                 false_easting=0, false_northing=0, globe=None):
+                 false_easting=0, false_northing=0, globe=None, sweep_axis='y'):
```

Comment:
Yeah, in my logic before, I missed the fact that the best reason to use a default of `'y'` is that it's identical to what was used before.

On my read of that paragraph, it seems consistent. Most importantly:
1. "Thus, the scanning geometry of the Meteosat series satellite should take sweep as x, and GOES should take sweep as y."
2. The files I have for GOES-16 specify `sweep_angle_axis` as `'x'`, and using this value makes the image line up perfectly with maps.